README.md

```diff
@@ -198,6 +198,98 @@ To add copyright headers to all files with IntelliJ
 2. Create a new copyright profile matching existing copyright header found on all files, name it Dockstore (Settings -> Copyright -> Copyright Profiles -> Add New)
 3. Set the default project copyright to Dockstore (Settings -> Copyright)
 
+### Setting up a Mac with Apple silicon for Dockstore development
+Install IntelliJ (Be sure to click on the dropdown and select '.dmg (Apple Silicon)')
+https://www.jetbrains.com/idea/download/#section=mac
+
+Add the Scala plugin to IntelliJ
+https://www.jetbrains.com/help/idea/managing-plugins.html
+
+Install Docker (Be sure to click on 'Mac with Apple chip')
+https://docs.docker.com/desktop/mac/install/
+
+Install Brew
+https://brew.sh/
+```
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Run 'git' to trigger the install of Xcode or the Command Line Tools which will install and or update git
+https://developer.apple.com/forums/thread/672087?answerId=659036022#659036022
+```
+git
+```
+_(If that doesn't work install git manually https://git-scm.com/download/mac)_
+
+
+Setup git user information
+```
+git config --global user.email "you@example.com"
+git config --global user.name "Your Name"
+```
+[Read about git token requirements](https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/)
+
+[Setup personal access token for git CLI](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
+
+[It's helpful to cache your git personal access token](https://docs.github.com/en/get-started/getting-started-with-git/caching-your-github-credentials-in-git)
+
+Install Hubflow
+https://datasift.github.io/gitflow/TheHubFlowTools.html
+```
+git clone https://github.com/datasift/gitflow
+cd gitflow
+sudo ./install.sh
+```
+
+Install JDK 17
```

Comment:
This is really thorough, but I wonder if this will tend to get out-of-date beyond this point. 
On option may be to do this in a github action so we know when it breaks and then link to that action
https://github.com/actions/virtual-environments#available-environments

This is obviously out-of-scope, but should be a follow-up ticket.

Reply:
![Screen Shot 2022-06-17 at 3 55 57 PM](https://user-images.githubusercontent.com/3650921/174410518-4987445f-9df4-42f6-aa84-a7968c4717a1.png)
![Screen Shot 2022-06-17 at 3 56 15 PM](https://user-images.githubusercontent.com/3650921/174410520-936c62fc-e422-4a21-a59c-1ff8be81b299.png)
![Screen Shot 2022-06-17 at 3 56 26 PM](https://user-images.githubusercontent.com/3650921/174410521-319ac4e4-ad3b-42bf-bd35-ff5a8db15157.png)

Reply:
Created ticket https://ucsc-cgl.atlassian.net/browse/SEAB-4529 to use GitHub Action